web/js/components/ReadSupers.jsx

```diff
@@ -0,0 +1,79 @@
+import * as React from 'react';
+import * as ReactDOM from 'react-dom';
+import Superlative from './Superlative.jsx'
+import Notifications, {notify} from 'react-notify-toast';
+
+// Hard-coding supers for testing 
+var supers = [
+  {
+  "name":"Most likely to die in a freak accident",
+  "created_by":"A",
+  "assigned_by":"B",
+  "receipient":"C"
+  },
+  {
+  "name":"Most loyal",
+  "created_by":"D",
+  "assigned_by":"E",
+  "receipient":"F"
+  },
+  {
+  "name":"Best hair",
+  "created_by":"D",
+  "assigned_by":"E",
+  "receipient":"F"
+  },
+  {
+  "name":"Most likely to become president",
+  "created_by":"D",
+  "assigned_by":"E",
+  "receipient":"F"
+  }
+]
+
+export default class ReadSupers extends React.Component {
+  constructor(props){
+    super(props);
+
+    this.state = {
+      currentSuper:"",
+      supersList:[]
+    }
+
+    this.nextButtonClicked = this.nextButtonClicked.bind(this)
+
+  }
+  render() {
+    return (
+      <div>
+        <Notifications />
+        <Superlative name={this.state.currentSuper}/>
```

Comment:
rather than having this value get calculated in setState, you should get the first element here

Reply:
good idea! updating